Frontend/src/ui/Tabs.jsx

```diff
@@ -74,7 +75,7 @@ export default function PageTabs() {
         render: () => {},
       },
     ]
-  }, [competitionInfo.id, navigate])
+  }, [canAdminCompetition, competitionInfo.id, navigate])
```

Comment:
I think `canAdminCompetition` will be 'new' every render since it's a function (unless it's memoized somewhere?), so this `useMemo` is running every time.
I'm not sure if adding `canAdminCompetition(competitionInfo.id)` to the dependency array instead is allowed.

Reply:
eslint was the one to warn me about adding it to the dependency array, but I will investigate if it does

Reply:
not relevant any more now that this is a boolean